### PR TITLE
Adding implementation for Hedera ID Change backwards compatibility.

### LINKED ISSUE #633: Errors caused by Hedera ID change
After the implementation of the Replace Hedera network ID based on HIP-26 #591, we will face a situation where we will have old messages with Hedera ID = 0 and new messages with Hedera ID = 296. 

To reproduce the error you will need to:
- have a read-only validator 
- have the new Hedera ID = 296 in the config 
- Process an old message with Hedera ID = 0  (for example run the validator from the beginning)

Expected problems: Messages will not be processed, because they will be with 0 instead of 296 from EVM to 0. Events with unlocking, mint will throw an error.

Keep in mind that the network ID is stored also in the DB `transfers` table: `source_chain_id`, `target_chain_id`, `native_chain_id`

Whenever we display (Example /transfer-history) 0 for Hedera network ID, replace it with 296 (the new id from the config)

### PR DESCRIPTION
**Pull request by Psykepro**
Signed-off-by: Nikolay Nedkov <nikolay.nedkov@limechain.tech>

**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #633

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: config/bridge.yml [hunk 1]
```diff
@@ -1,25 +1,320 @@
-# This file contains application defaults
```
Comment (svetlio8) [inline config/bridge.yml:1]: commit with the cofig

File: config/node.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#DAVE read-only
```
Comment (svetlio8) [inline config/node.yml:1]: commit with the cofig

File: e2e/setup/application.yml [hunk 3]
```diff
@@ -1,44 +1,57 @@
 #E2E ETH-Hedera
```
Comment (svetlio8) [inline e2e/setup/application.yml:1]: commit with the cofig

File: e2e/setup/bridge.yml [hunk 4]
```diff
@@ -1,22 +1,69 @@
 # This file contains application defaults
```
Comment (svetlio8) [inline e2e/setup/bridge.yml:1]: commit with the cofig

File: examples/three-validators/alice/config/node.yml [hunk 5]
```diff
@@ -1,13 +1,41 @@
 # This file contains application defaults
```
Comment (svetlio8) [inline examples/three-validators/alice/config/node.yml:1]: commit with the cofig

File: examples/three-validators/bob/config/node.yml [hunk 6]
```diff
@@ -1,13 +1,41 @@
 # This file contains application defaults
```
Comment (svetlio8) [inline examples/three-validators/bob/config/node.yml:1]: commit with the cofig

File: examples/three-validators/bridge.yml [hunk 7]
```diff
@@ -1,25 +1,69 @@
 # This file contains application defaults
 # Check docs/configuration.md for more information
-#bridge:
```
Comment (svetlio8) [inline examples/three-validators/bridge.yml:3]: commit with the cofig

File: examples/three-validators/carol/config/node.yml [hunk 8]
```diff
@@ -1,13 +1,41 @@
 # This file contains application defaults
 # Check docs/configuration.md for more information
-#node:
```
Comment (svetlio8) [inline examples/three-validators/carol/config/node.yml:3]: commit with the cofig

**Review by svetlio8 — CHANGES_REQUESTED** [review]
Please remove the committed configs

**Comment by failfmi** [comment]
You have not merged `main` properly. Things that have been committed in main show as changes..

**Review by svetlio8 — APPROVED** [review]
nit: please check e2e

**Review by vectorman1 — APPROVED** [review]

**Pull request merged**